src/components/UI/Icon/Icon.tsx

```diff
@@ -148,25 +148,25 @@ export const IconResolverAddress: React.FunctionComponent<SVGProps<SVGElement>>
   );
 };
 
```

Comment:
create storybook views for these?

Reply:
done. create storybook views for all the icons.